Sources/Sentry/SentryPerformanceTracker.m

```diff
@@ -213,7 +215,12 @@ - (void)finishSpan:(SentrySpanId *)spanId withStatus:(SentrySpanStatus)status
     id<SentrySpan> spanTracker;
     @synchronized(self.spans) {
         spanTracker = self.spans[spanId];
-        [self.spans removeObjectForKey:spanId];
+        // Hold reference for tracer for longer because automatic
+        // tracer aren't  being referenced by anything else.
+        // callback to `tracerDidFinish` will release it.
+        if (![spanTracker isKindOfClass:SentryTracer.self]) {
```

Comment:
`m`: Why do we need this change? What's the impact?

Reply:
We need to keep a reference to SentryTracer otherwise ARC will release it.
As mentioned in the comment, tracerDidFinish will be responsable for releasing the Tracers from now on.